class/defaults.yml

```diff
@@ -41,7 +41,7 @@ parameters:
     name: keycloak
     charts:
       keycloakx: "1.3.2"
-      postgresql: "10.16.2"
+      postgresql: "11.6.15"
```

Comment:
What's the reason to update to 11 and not to 12 or even 13 or 14? 

I recently searched for compatibility for keycloak and dind't fin a clear statement which version should be used.

Reply:
That's the helm chart version, not the Postgres version.